MAUI/Cartesian-Charts/Annotation.md

```diff
@@ -0,0 +1,743 @@
+
+        </chart:SfCartesianChart>
+
+  {% endhighlight %}
+
+{% highlight c# %}
+    
+        SfCartesianChart chart = new SfCartesianChart();
+        
+        var rectangle = new RectangleAnnotation()
+        {
+            X1 = 10,
+            Y1 = 105,
+            X2 = 300,
+            Y2 = 200,
+            CoordinateUnit = ChartCoordinateUnit.Pixel,
+            Text = "Rectangle"
+        };
+
+        chart.Annotations.Add(rectangle);
+
+        this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Annotation in pixel value in MAUI Chart](Annotation_image/rectangle_pixel_value.png)
+
+## Adding annotation for multiple axes
+
+When there are multiple axes, annotations can also be added to a particular axis by using the [XAxisName]() and [YAxisName]() properties. This can be shown in the code snippet below:
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+
+    <chart:SfCartesianChart.YAxes>
+        <chart:NumericalAxis Minimum="0" Maximum="1"/>
+        <chart:NumericalAxis Name="YAxis"/>
+    </chart:SfCartesianChart.YAxes>
+
+    <chart:SfCartesianChart.Annotations>
+
+        <chart:EllipseAnnotation X1="2" X2="4" Y1="10" Y2="15" Text="Ellipse" YAxisName="YAxis"/>
+
+    </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+ {% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    NumericalAxis yAxis1 = new NumericalAxis();
+    yAxis1.Minimum = 0;
+    yAxis1.Maximum = 1;
+    chart.XAxes.Add(yAxis1);
+    NumericalAxis yAxis2 = new NumericalAxis();
+    yAxis2.Name = "YAxis";
+    yAxis2.CrossesAt = double.MaxValue;
+    chart.YAxes.Add(yAxis2);
+
+    var ellipse = new EllipseAnnotation()
+    {
+        X1 = 2,
+        Y1 = 10,
+        X2 = 4,
+        Y2 = 15,
+        Text = "Ellipse",
+        YAxisName = "YAxis"
+    };
+
+    chart.Annotations.Add(ellipse);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Annotation in multiple axes in MAUI Chart](Annotation_image/multiple_axes.png)
+
+## Text annotation
+
+The [TextAnnotation]() is used to add simple text at specific points in the chart area using the [Text]() property.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+
+        <chart:SfCartesianChart.Annotations>
+        
+            <chart:TextAnnotation X1="2" Y1="25" Text="TextT Annotation"/>
+            
+    </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+ {% endhighlight %}
+
+{% highlight c# %}
+
+        SfCartesianChart chart = new SfCartesianChart();
+
+        var text = new TextAnnotation()
+        {
+            X1 = 2,
+            Y1 = 25,
+            Text = "Text Annotation"    
+        };
+
+        chart.Annotations.Add(text);
+
+        this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Text annotation in MAUI Chart](Annotation_image/text_annotation.png)
+## Text Customization
```

Comment:
@NitheeshkumarThangaraj  ,I have provided the suggestion to use sentences as MAUI customization categories, like plot band and axis level, which we already discussed during the scrum call. Why is this not addressed?

Reply:
@SaiyathAliFathima , addressed the content as per in maui